packages/ui/styles/constants/_sizes.scss

```diff
@@ -5,7 +5,7 @@
 
 $grid: 0.25rem !default;
 $alertWidth: 350px !default;
-$borderRadius: $grid * 1.25 !default;
+$borderRadius: $grid * 2 !default;
```

Comment:
It will change a lot of other border-radiuses. Is @pawelklasa OK with that?

Reply:
It is adapted from the design (border-radius: 8px). Let's wait for a response from @pawelklasa

Reply:
Most probably he won't answer here, it's better to ask him in the https://hazelcast.slack.com/archives/C01BT11F24E

Reply:
Those look good to me.  @emrberk @fantkolja